dev-client/src/util.ts

```diff
@@ -0,0 +1,8 @@
+export function formatNames(firstName: string, lastName?: string) {
+  let formatted = firstName;
+  if (lastName !== undefined) {
+    formatted += ', ';
+    formatted += lastName;
+  }
+  return formatted;
```

Comment:
question: will this output "first, last" rather than "last, first"? `+=` for non-commutative addition operations is confusing :thinking:

Reply:
Yes, it will! I was not paying attention here.